project-ivi/ivi
# Integrate Ace Module

\#54

# Description
I found an open source javascript-based code editor called ace that can be easily integrated into our project. I am using the modules `brace` and `react-ace` which make ace browserify and react compatible, respectively.

# Change Log
`package-lock.json` `package.json` : Install `brace` and `react-ace` modules.
`editor/index.js` : Swap about semantic `TextArea` for the `AceEditor` component.

# Manual Test Steps
Try writing some code in the editor pane. Make sure syntax highlighting seems to work correctly, and that the editor seems to display expected, intuitive behavior in any situation.
